June assets (#657)
akeneo/pim-helpcenter: content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md

> # Manually execute naming conventions on assets

Reviewer: ```suggestion
# New: execute asset naming conventions manually
```

> ::: meta-data type="New" features="Asset Manager" available="late-June" link-to-doc="../articles/assets-product-link-rules.html#focus-on-the-naming-convention.html"

Reviewer 2: It's either "early June", "mid-June" or "late June".

:::

> In the Asset Manager, it is possible to define a naming convention in order to automatically populate asset attributes with values contained in the asset filename or asset code. This naming convention is executed at the asset creation.

Reviewer 2: To rephrase. We need the new feature/improvement to be in the very first sentence as this is the one we will show in the communication panel of the PIM.

> In the Asset Manager, it is possible to define a naming convention in order to automatically populate asset attributes with values contained in the asset filename or asset code. This naming convention is executed at the asset creation.

Reviewer: ```suggestion
You can now manually execute naming conventions from the user interface whenever you need it. This new feature can be very useful if your naming convention has changed after asset uploads or creations.

Asset naming conventions populate asset attributes with values contained in the asset filename or asset code, prior to this improvement, naming conventions were executed once at the asset creation. 
```

> From now on, you can manually re-execute this naming convention afterward. It can be very useful if your naming convention changes after the asset upload/creation for instance.

Reviewer 3: "This naming convention was executed at the asset creation but from now on, you can also manually re-execute this naming convention afterward!"

> From now on, you can manually re-execute this naming convention afterward. It can be very useful if your naming convention changes after the asset upload/creation for instance.

Reviewer: ```suggestion
```
Moved at the beginning of the article

> You have 2 ways to manually execute the naming convention:

Reviewer: ```suggestion
You have two ways to manually execute a naming convention:
```

> - either in each asset edit form (it's a unitary action that saves the asset and execute the naming convention)

Reviewer 3: "and executes"

> - either in each asset edit form (it's a unitary action that saves the asset and execute the naming convention)

Reviewer: ```suggestion
- for one specific asset, directly in the asset edit form. The action is visible under the `...`
```

![Unitary execution](../img/unitary-execution.png)

> - or in the `product link rules` tab of your asset family (it's a mass action that executes the naming convention on all the assets of this asset family).

Reviewer: ```suggestion
- for an asset family, under the `product link rules` tab of your asset family - the naming convention will be applied to all assets belonging to this family.
```

![Mass execution](../img/mass-execution.png)

::: more

> [About the product link rules](../articles/assets-product-link-rules.html)  

Reviewer 2: Each article link should be formulated as a question.

> [About the product link rules](../articles/assets-product-link-rules.html)  

Reviewer: ```suggestion
[What is a product link rule?](../articles/assets-product-link-rules.html)  
```

> [Create and display assets](../articles/create-and-display-assets.html)  

Reviewer 2: Each article link should be formulated as a question.

> [Create and display assets](../articles/create-and-display-assets.html)  

Reviewer: ```suggestion
[How to create and display assets?](../articles/create-and-display-assets.html)  
```

> [Manage your asset families](../articles/manage-asset-families.html)

Reviewer 2: Each article link should be formulated as a question.

> [Manage your asset families](../articles/manage-asset-families.html)

Reviewer: ```suggestion
[How to manage your asset families?](../articles/manage-asset-families.html)
```

# New: execute asset naming conventions manually
::: meta-data type="New" features="Asset Manager" available="late June" link-to-doc="../articles/assets-product-link-rules.html#focus-on-the-naming-convention.html"
:::

You can now manually execute naming conventions from the user interface whenever you need it. This new feature can be very useful if your naming convention has changed after asset uploads or creations.

> Asset naming conventions populate asset attributes with values contained in the asset filename or asset code, prior to this improvement, naming conventions were executed once at the asset creation. 

Reviewer 4: ```suggestion
Asset naming conventions populate asset attributes with values contained in the asset filename or asset code, prior to this improvement, naming conventions were executed once at the asset creation step. 
```

> Asset naming conventions populate asset attributes with values contained in the asset filename or asset code, prior to this improvement, naming conventions were executed once at the asset creation step. 

Reviewer 2: ```suggestion
Asset naming conventions populate asset attributes with values contained in the asset filename or asset code. Prior to this improvement, naming conventions were executed once at the asset creation step. 
```

> You have two ways to manually execute a naming convention:

Reviewer 4: ```suggestion
There are two ways to manually execute a naming convention:
```
